plugins/Homologues.py

```diff
@@ -118,7 +125,55 @@ def process(self, conf, output, cmd_conf):
                     download_manifest.msg_completion = \
                         f"Homologue '{species}' data, JQ filtered with '{conf.jq}'"
                     download_manifest.status_completion = ManifestStatus.COMPLETED
-            manifest_step.resources.append(download_manifest)
+            mapping_data_manifest.resources.append(download_manifest)
+        return mapping_data_manifest
+
+    def download_homology_data(self, conf, output, download) -> List[ManifestResource]:
+        """
+        Download the homology data for the species of interest.
+
+        Files look like:
+            Compara.<release>.[type]+_default.homologies.tsv.gz -> <type>-<species>.tsv.gz
```

Comment:
There's probably a typo here 'Compara.<release>.[type]+_default.homologies.tsv.gz' I think it is 'Compara.<release>.[type]_default.homologies.tsv.gz'